jboss/container/wildfly/launch/logger-category/added/launch/configure_logger_category.sh

```diff
@@ -46,29 +48,30 @@ add_logger_category() {
               local ret
               testXpathExpression "${xpath}" "ret"
               if [ "${ret}" -ne 0 ]; then
-                echo "You have set LOGGER_CATEGORIES to configure a logger. Fix your configuration to contain the logging subsystem for this to happen." >> ${CONFIG_ERROR_FILE}
+                echo "You have set LOGGER_CATEGORIES to configure a logger. Fix your configuration to contain the logging subsystem for this to happen." >> "${CONFIG_ERROR_FILE}"
               fi
         fi
 
         log_info "Found env LOGGER_CATEGORIES, configuring...."
         for i in ${LOGGER_CATEGORIES}; do
             logger=$(echo "$i" | sed 's/ //g')
-            logger_category=$(echo $logger | awk -F':' '{print $1}')
-            logger_level=$(echo $logger | awk -F':' '{print $2}')
+            logger_category=$(echo "$logger" | awk -F':' '{print $1}')
+            logger_level=$(echo "$logger" | awk -F':' '{print $2}')
+            allowed_log_levels_str=$(printf ",%s", "${allowed_log_levels[@]}")
```

Comment:
Note this is a manual change, converting an array to a string seperated with ','